Source/WebInspectorUI/UserInterface/Controllers/JavaScriptRuntimeCompletionProvider.js

```diff
@@ -91,6 +91,19 @@ WI.JavaScriptRuntimeCompletionProvider = class JavaScriptRuntimeCompletionProvid
         return JavaScriptRuntimeCompletionProvider.__cachedCommandLineAPIKeys;
     }
 
+    // Public
+
+    // Clears the cache of property names to force picking up any changes occurred while stepping, executing new code in the console prompt, or sitting idle.
```

Comment:
Style: the comment should be inside the function

also, most of this depends more on how it's used, not this code itself, so if you're determined to keep the comment id probably move it to the various callsites instead (and my preference would be to drop it entirely since the name of the function should be clear enough to explain what it's for, and if not we should make the name clearer)